.test-infra/jenkins/PrecommitJobBuilder.groovy

```diff
@@ -116,7 +119,7 @@ class PrecommitJobBuilder {
       steps {
         gradle {
           rootBuildScriptDir(commonJobProperties.checkoutDir)
-          tasks(gradleTask)
+          tasks(gradleTasks + (gradleTask == null ? []: [gradleTask]))
```

Comment:
Note that this change is in a separate commit that should be kept separate for clarity of code history.